### ISSUE TITLE
Input pre-processing for s3dg, i3d, d3d

### ISSUE THREAD

**Issue by rahman-mdatiqur** [princeton-vl/d3dhelper#3]
Hello @jonathancstroud ,

The input pre-processing that you have used in the notebook is im/255 which re-scales the values in the range [0,1]. But the original i3d paper from deepmind have a different input pre-processing for the rgb and flow inputs that re-scales the values in the range [-1,1]. 

So, I was just wondering, for the s3dg, i3d and d3d models that you made available in the [GoogleDrive](https://drive.google.com/drive/folders/1Yb-g-Ae_B4tyM7N7bk0kRd2r5ChO-oa3?usp=sharing), what input pre-processing should I use for rgb and flow inputs, [0,1] or [-1,1] ? 

I would highly appreciate your feedback on this.

Thanks.

**Comment by jonathancstroud** [comment]
Hi, thanks for pointing this out. Looks like the code is incorrect, it should be [-1, 1]. I'm not able to test this change at the moment, so please let me know if this works.

**Comment by rahman-mdatiqur** [comment]
Hello @jonathancstroud ,

I have tested the checkpoints with both types of input pre-processing. I found that the results are similar. But, to confirm, it should be [-1,1] for the provided checkpoints, right?

**Comment by rahman-mdatiqur** [comment]
Hello @jonathancstroud 
One more question. For d3d checkpoint, how do I load the model? Do I simply use the s3dg network in rgb mode to load d3d? If so, then what's the basic difference between s3dg and d3d?

I would highly appreciate your response.

**Comment by jonathancstroud** [comment]
Yes, I confirm that it should be [-1, 1]. You can see an example of correct preprocessing (for images) [here](https://github.com/tensorflow/models/blob/master/research/slim/preprocessing/inception_preprocessing.py). The I3D/S3D-G/D3D networks are all based on Inception-v1, so you can apply the Inception input image preprocessing frame-by-frame to get good results for these models. Glad to hear the results are similar!

For your second question: yes, you use the S3D-G network in RGB mode to load D3D. D3D has the same model architecture as S3D-G, but is trained differently. For more details on the training procedure, please refer to the [arxiv paper](https://arxiv.org/abs/1812.08249). Conceptually, D3D can have any backbone network, but we use S3D-G as the backbone, so we simply use "D3D" to refer to "D3D with an S3D-G backbone".

Hope that helps!

**Comment by rahman-mdatiqur** [comment]
Dear  @jonathancstroud 
Thanks a lot for your detailed and very useful feedback. I highly appreciate it.

Just wondering, if you have any plan to release the training code for d3d.
